tests/test_api.py

```diff
@@ -36,6 +38,14 @@ def test_upload_image_default_domain() -> None:
     assert open(IMG_LOC, "rb") == con.content
 
 
+@pytest.mark.slow
+def test_upload_with_io_bytes():
```

Comment:
I didn't add:
```py
@pytest.mark.xfail(run=False, reason="sxcu optimises images")
```
because this is a 1x1 white square and I assume sxcu can't / won't optimize that

Reply:
I guess that's fine and tests didn't fail